CONTRIBUTING.md

```diff
@@ -1,23 +1,48 @@
 # How To Contribute To Beaker
 
+Contributions are welcomed. Simple bug fixes and minor enhancements will usually be accepted. Larger features should be discussed with a team member before you invest in developing them with the expectation that they will be merged.
+
+## Recommended Tools
+
+It may be necessary to test Beaker against multiple versions of Ruby. The maintainers use [`rbenv`](https://github.com/rbenv/rbenv) to manage multiple Ruby versions; you can install it with Homebrew. You'll also want [`rbenv-bundler`](https://github.com/carsomyr/rbenv-bundler) to keep Gem dependencies from conflicting.
+
 ## Getting Started
 
+Beaker does not use GitHub Issues, but an internal ticketing system running Jira that interfaces with other services. To be accepted by the maintainers, changes must follow this workflow and tagging scheme. See [ticket process doc](docs/concepts/ticket_process.md) for a
+
 * Create a [Jira account](http://tickets.puppetlabs.com)
 * Make sure you have a [GitHub account](https://github.com/signup/free)
-* Submit a ticket for your issue, assuming one does not already exist.
+* Submit a ticket for your issue on Jira, assuming one does not already exist.
   * Clearly describe the issue including steps to reproduce when it is a bug.
-  * File a ticket in the [BKR project](https://tickets.puppetlabs.com/issues/?jql=project%20%3D%20BKR)
+  * File the ticket in the [BKR project](https://tickets.puppetlabs.com/issues/?jql=project%20%3D%20BKR)
 * Fork the [Beaker repository on GitHub](https://github.com/puppetlabs/beaker)
 
-## Making Changes
+## Installation/Setup for Development
```

Comment:
please move installation to the installation doc. This goes for the "Recommended Tools" section above as well.

The contributor guide should be a resource for people to come back to over & over again as they go through their time contributing to beaker, so we shouldn't have one-time things here.

With that in mind, the Getting Started section should really be about Filing Bugs & the one-time JIRA creation steps can be moved to the ticket process doc.